shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -63,6 +97,11 @@ @interface FlutterViewController () <FlutterBinaryMessenger,
 @property(nonatomic, assign) BOOL isHomeIndicatorHidden;
 @property(nonatomic, assign) BOOL isPresentingViewControllerAnimating;
 
+/**
+ * Whether the device is rotating.
```

Comment:
nits: by looking at the property name and document, one might get confused thinking this bool indicates whether the physical device is rotating.